pkg/gatewayconfigurationserver/gatewayconfigurationserver_test.go

```diff
@@ -46,6 +46,19 @@ var (
 	}
 )
 
+func TestGatewayConfigurationServer(t *testing.T) {
+	ctx := log.NewContext(test.Context(), test.GetLogger(t))
+
+	conf := &component.Config{}
+	c := component.MustNew(test.GetLogger(t), conf)
+
+	test.Must(New(c, testConfig))
+	test.Must(c.Start(), nil)
+	defer c.Close()
+
+	mustHavePeer(ctx, c, ttnpb.ClusterRole_GATEWAY_CONFIGURATION_SERVER)
```

Comment:
This test function contains no assertions and only panics on errors.
Would be nicer/cleaner/easier to debug if there were actual assertions here.

Reply:
This was just copied from `TestDeviceTemplateConverter`, and I guess it's done this way in more places, so if this is something you think is important, please file a technical debt issue.